code/mob/living/carbon/human.dm

```diff
@@ -2563,7 +2563,7 @@
 			src.handcuffs.destroy_handcuffs(src)
 		else
 			src.last_resist = world.time + 100
-			var/calcTime = src.handcuffs.material ? max((src.handcuffs.material.getProperty("hard") + src.handcuffs.material.getProperty("density")) * 10, 200) : (istype(src.handcuffs, /obj/item/handcuffs/guardbot) ? rand(150, 180) : (src.canmove ? rand(400,500) : rand(600,750)))
+			var/calcTime = istype(src.handcuffs, /obj/item/handcuffs/guardbot) ? rand(150, 180) : (src.canmove ? rand(400,500) : rand(600,750))
```

Comment:
Would rather just clamp max cuff time to the maximum possible non-material cuff timer
```suggestion
			var/calcTime = src.handcuffs.material ? clamp((src.handcuffs.material.getProperty("hard") + src.handcuffs.material.getProperty("density")) * 10, 200, rand(600, 750)) : (istype(src.handcuffs, /obj/item/handcuffs/guardbot) ? rand(150, 180) : (src.canmove ? rand(400,500) : rand(600,750)))
```